nntrainer/utils/node_exporter.cpp

```diff
@@ -99,6 +114,76 @@ void Exporter::saveTflResult(const std::tuple<props::Unit> &props,
   tf_node->setBuiltinOptions(tflite::BuiltinOptions_FullyConnectedOptions,
                              flatbuffers::Offset<void>());
 }
+
+template <>
+void Exporter::saveTflResult(const std::tuple<props::Activation> &props,
+                             const ActivationLayer *self) {
+  createIfNull(tf_node);
+
+  tf_node->setOpType(tflite::BuiltinOperator_RELU);
+  /// we probably going to need flatbuffer inside exporter regarding this
+  tf_node->setBuiltinOptions(tflite::BuiltinOptions_NONE,
+                             flatbuffers::Offset<void>());
+}
+
+template <>
+void Exporter::saveTflResult(
+  const std::tuple<props::FilterSize, std::array<props::KernelSize, 2>,
+                   std::array<props::Stride, 2>, props::Padding2D> &props,
+  const Conv2DLayer *self) {
+  createIfNull(tf_node);
+
+  auto weight_transform = [](std::vector<const Tensor *> &weights) {
+    std::vector<Tensor> new_weights;
+
+    Tensor filter(*weights[0]);
+    // TODO support 4 rank tranpose
+    // weights[0]->transpose("2:3:1:0")
+    convertNCHWToNHWC(*weights[0], filter);
+    Tensor bias(*weights[1]);
+    convertNCHWToNHWC(*weights[1], bias);
+
+    new_weights.push_back(filter);
+    new_weights.push_back(bias);
+
+    return new_weights;
+  };
+  tf_node->setWeightTransformFn(weight_transform);
+
+  tf_node->setOpType(tflite::BuiltinOperator_CONV_2D);
+  /// we probably going to need flatbuffer inside exporter regarding this
+  tf_node->setBuiltinOptions(tflite::BuiltinOptions_Conv2DOptions,
+                             flatbuffers::Offset<void>());
+}
+
+template <>
+void Exporter::saveTflResult(
+  const std::tuple<props::Normalization, props::Standardization> &props,
+  const InputLayer *self) {
+  createIfNull(tf_node);
+  // input layer exports to Transpose operator (NCHW -> NHWC)
+  tf_node->setOpType(tflite::BuiltinOperator_TRANSPOSE);
+  /// we probably going to need flatbuffer inside exporter regarding this
+  tf_node->setBuiltinOptions(tflite::BuiltinOptions_TransposeOptions,
+                             flatbuffers::Offset<void>());
+
+  auto input_transform = [](std::vector<const Tensor *> &inputs) {
+    std::vector<Tensor> new_inputs;
+    assert(inputs.size() == 1);
+    new_inputs.reserve(inputs.size() + 1 /* perm */);
```

Comment:
line comments should begin with /**. you can check the ci http://ci.nnstreamer.ai/nntrainer/ci/repo-workers/pr-checker/1892-202204270116260.84354090690613-3b2dfca6b9da817877edc232fb1419dea815df9a/report/doxygen_tag.txt